packages/styles/scss/globals/utils/_vendor-prefix.scss

```diff
@@ -0,0 +1,22 @@
+/// Mixin to prefix several properties at once
+/// @author Hugo Giraudel
+/// @param {Map} $declarations - Declarations to prefix
+/// @param {List} $prefixes (()) - List of prefixes to print
+/// @example scss
+/// .selector {
+///   @include prefix((
+///     column-count: 3,
+///     column-gap: 1.5em,
+///     column-rule: 2px solid hotpink
+///   ), webkit moz);
+/// }
+@mixin prefix($declarations, $prefixes: ()) {
```

Comment:
@kennylam I'm, wondering if we should introduce an autoprefixer in gulp instead?

Reply:
@kennylam actually it looks like we're already using autoprefixer, but I'm wondering if it's how it's configured that we're missing iOS12:

https://github.com/carbon-design-system/ibm-dotcom-library/blob/989093d00ab2ca8f723d4b9530e3162e2d7d0481/packages/styles/gulp/tasks/sass.js#L33

Reply:
@jeffchew I considered this also but decided against it because I feel like we should allow this to be on a as-needed basis. The latest browsers no longer need vendor prefixes at all and we probably don't need to add prefixes to everything by default. What do you think?

Reply:
well at least for iOS Safari 12, I'm seeing why it's possibly causing an issue:

https://browserl.ist/?q=%3E+1%25%2C+Last+2+versions

Based on this, Safari on iOS 12.0-12.1 isn't included, and would probably see this issue. Should verify what version of safari is being used in browserstack.

But I hear you, it's a tactical fix just for this instance too so the risk is super low. I think I'm ok with pushing this specific fix.

Reply:
As discussed, @kennylam is looking into adding autoprefixer in storybook's webpack configuration to see if that fixes the issue.